docs/content/preview/develop/best-practices-ysql.md

```diff
@@ -179,6 +178,10 @@ YugabyteDB [smart drivers](../../drivers-orms/smart-drivers/) provide advanced c
 For more information, see [Load balancing with smart drivers](https://www.yugabyte.com/blog/multi-region-database-deployment-best-practices/#load-balancing-with-smart-driver).
 {{</lead>}}
 
+## Make sure the application uses new nodes
+
+When a cluster is expanded, newly added nodes do not automatically start to receive client traffic. Regardless of the driver type, the application must either explicitly request new connections or, if it is using a pooling solution, it can configure the pooler to recycle connections periodically (for example, by setting maxLifetime and/or idleTimeout).
```

Comment:
`Regardless of the driver type` - Do you mean `Regardless of the language of the driver`?

Reply:
@ashetkar Well also whether it is smart driver or normal driver